src/sinol_make/commands/run/__init__.py

```diff
@@ -753,11 +845,47 @@ def run(self, args):
             print(util.warning("WARN: Scores sum up to %d (instead of 100)." % total_score))
         print()
 
+        checker = glob.glob(os.path.join(os.getcwd(), "prog", f'{self.ID}chk.*'))
+        if len(checker) != 0:
+            print(util.info("Checker found. Running correct solution first."))
```

Comment:
Is there a reason for running first the correct solution only when there is a checker? I can't see the scenario where it's needed.

Reply:
The file with a correct answer is one of the arguments given to checker, so first the program has to generate them

Reply:
Isn't that the job of `make outgen`?

Reply:
Yes it is, I'll change it